sys/include/net/gnrc/netif.h

```diff
@@ -392,8 +392,8 @@ gnrc_netif_t *gnrc_netif_get_by_pid(kernel_pid_t pid);
  *                      @p CONFIG_GNRC_NETIF_IPV6_ADDRS_NUMOF `* sizeof(ipv6_addr_t)
  *                      here (and have @p addrs of the according length).
  *
- * @return  Number of addresses in @p addrs times `sizeof(ipv6_addr_t)` on
- *          success (including 0).
+ * @return  Size of addresses in @p addrs (number of addresses times `sizeof(ipv6_addr_t)`)
```

Comment:
```suggestion
 * @return  Size of the array of addresses in @p addrs (number of addresses times `sizeof(ipv6_addr_t)`)
```

? Size of addresses IMHO is ambiguous (could also just mean `sizeof(ipv6_addr_t)`)

Reply:
Though, on the other hand, "Size of the array of addresses" could also be misunderstood as `ARRAY_SIZE(addresses)` mhhhh :-/

Reply:
But maybe, with the clarifying statement in parentheses this is enough now.